data/blog/linux-32bit-support.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Discontinuing support for 32-bit Linux
+author: felixrieseberg
+date: '2019-03-15'
+---
+
+The Electron team will discontinue support for 32-bit/i386 Linux starting with Electron v4.0. The last version of Electron that supports 32-bit based installations of Linux is Electron v3.1, which will receive support releases until Electron v6 is released. We will continue to support 64-bit based Linux (x64).
```

Comment:
For reference, because I have to deal with this a lot in tooling:

* 32-bit: Node/Electron use `ia32`, Debian-based distros use `i386`, RPM-based distros use `i586` or `i686`.
* 64-bit: Node/Electron use `x64`, Debian-based distros use `amd64`, RPM-based distros use `x86_64`.

Reply:
Basically, what I'm saying is that we need to be consistent with which technology's terminology we use. I would suggest using both Node and Debian verbiage, because Ubuntu has the user share, and because we use Node terms in the tooling.

Reply:
@malept should we just drop the `(x64)` parenthetical in this paragraph?

Reply:
That might be the easiest thing to do.

Reply:
There's also the `i386` reference. The other option is to just replace `i386` with `ia32`.